integrationtests/connectiontester.go

```diff
@@ -9,12 +9,10 @@ import (
 // TestConnection used to test connection
 func TestConnection(addr string, port string) bool {
 	connected := false
-	for t := time.Now(); time.Since(t) < 3*time.Minute; {
-		con, err := net.DialTimeout("tcp", net.JoinHostPort(addr, port), time.Second*12)
-		if err == nil {
-			connected = true
-		}
-		con.Close()
+	con, err := net.DialTimeout("tcp", net.JoinHostPort(addr, port), time.Second*12)
```

Comment:
why did this block change?

Reply:
the test was talking so much time it waits for 3 minutes